cmd/edgectl/aes_install_errors.go

```diff
@@ -213,26 +215,29 @@ func (i *Installer) KnownLocalClusterResult() Result {
 
 // Unable to provision a load balancer (failed to retrieve the IP address)
 func (i *Installer) LoadBalancerError(err error) Result {
+	url := "https://www.getambassador.io/docs/latest/topics/install/help/load-balancer"
 	message := noTlsSuccess
 	message += "\n\n"
-	message += `We timed out waiting for the load balancer's IP address for the AES Service.
-- If a load balancer IP address shows up, simply run the installer again.
-- If your cluster doesn't support load balancers, you'll need to expose AES some other way.`
+	message += fmt.Sprintf(`We timed out waiting for the load balancer's IP address for the AES Service.
+Visit %v for a more detailed explanation and step-by-step instructions about exposing a public load balancer to complete the Ambassador Edge Stack installation.`, url)
 	message += "\n\n"
 
 	return Result{
-
 		Report:  "fail_loadbalancer_timeout",
```

Comment:
No ShortMessage here?

Reply:
Shouldn't it be "We timed out waiting for the load balancer's IP address for the AES Service"?